src/main/java/com/robinpowered/sdk/model/DeviceManifest.java

```diff
@@ -52,7 +52,7 @@ public DeviceManifest(int id, String name, String slug, Boolean isRelay, String
 
 
     @Override
-    public int getId() {
```

Comment:
Is there a reason we're using a boxed generic here?
That signals to me that the method might return a null instance